apps/web/next.config.js

```diff
@@ -14,9 +24,6 @@ const nextConfig = {
 			'apistage.gauzy.co',
 			'gauzy.s3.wasabisys.com'
 		]
-	},
-	experimental: {
```

Comment:
@paradoxe35 why we remove that in this PR?

Reply:
In NextJS 13 this setting is not available
Since the AppDir is no longer experimental, it is already included in the module core

Also It could raise error while building image

@evereq